tmt/checks/enums.py

```diff
@@ -0,0 +1,28 @@
+import enum
```

Comment:
Why keep them in a standalone file? `__init__.py` has about 250+ lines, isn't that small enough?

Reply:
I just needed to deal with that circular import somehow. Would gladly do a less bad solution.